dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx

```diff
@@ -122,18 +87,25 @@ const usePayload = ({
 	section,
 	isPaidContent,
 	tags,
+	pageId,
+	keywordIds,
 }: {
 	shouldHideReaderRevenue: boolean;
 	section: string;
 	isPaidContent: boolean;
 	tags: TagType[];
-}) => {
-	const consent = useConsent();
+	pageId: string;
+	keywordIds: string;
+}): EpicPayload | undefined => {
+	const articleCounts = useArticleCounts(pageId, keywordIds);
```

Comment:
`useArticleCounts` is used instead of getting the data straight from localstorage